pyproject.toml

```diff
@@ -0,0 +1,88 @@
+[tool.poetry]
```

Comment:
Do we have poetry section even before integrating poetry ?

Reply:
These section headers are ignored by the pip interprator, it only parses thet packages, and their versions.

Removing them will make the eventual migration more cumbersome, which is why I left it.

Reply:
@caniko could you remove all poetry references so that this PR is only for migration to pyproject.toml? Even though they are ignored, we should keep all poetry-related stuff in the poetry PR ;)

Reply:
The issue is that the poetry format is very different from the native Python format. I will need to put aside some time, and do it in a few days. [Here](https://github.com/fastapi-users/fastapi-users/blob/master/pyproject.toml) is pure-python `pyproject.toml` example from `fastapi-users` repo.